CHANGELOG.md

```diff
@@ -13,6 +13,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 * `jj git push --deleted` will remove all locally deleted branches from the remote.
 
+* `jj restore` without `--from` works correctly even if `@` is a merge
+  commit. For now, there is an outstanding bug that conflicts may be restored
+  into a conflict that's presented differently than before (though the conflict
+  remains equivalent to what it was).
```

Comment:
The bit about an outstanding bug is stale, right?

Reply:
Yep. It's gone now.